code/datums/tips/tips.dm

```diff
@@ -0,0 +1,341 @@
+	messages = list(
+		"You can catch thrown items by toggling on your throw mode with an empty hand active.",
+		"To crack the safe in the vault, use a stethoscope on it.",
+		"You can climb onto a table by dragging yourself onto one.",
+		"You can drag other players onto yourself to open the strip menu, letting you remove their equipment or force them to wear something.",
+		"Clicking on a windoor rather then bumping into it will keep it open, you can click it again to close it.",
+		"You can spray a fire extinguisher or fire a gun while floating through space to change your direction. Simply fire opposite to where you want to go.",
+		"You can change the control scheme by pressing tab. One is WASD, the other is the arrow keys. Keep in mind that hotkeys are also changed with this.",
+		"All vending machines can be hacked to obtain some contraband items from them, and some can be fed with coins to gain access to premium items.",
+		"Glass shards can be welded to make glass, and metal rods can be welded to make metal.",
+		"If you need to drag multiple people either to safety or to space, bring a locker over and stuff them all in before hauling them off.",
+		"You can grab someone by clicking on them with the grab intent, then upgrade the grab by clicking on the grab button in your active hand. An aggressive grab will allow you to place someone on a table by clicking on it, or throw them by toggling on throwing.",
+		"Holding alt and left clicking a tile will allow you to see its contents in the top right window pane, which is much faster than right clicking.",
+		"The resist button will allow you to resist out of handcuffs, being buckled to a chair or bed, out of locked lockers and more. Whenever you're stuck, try resisting!",
+		"You can move an item out of the way by dragging it and then clicking on an adjacent tile with an empty hand.",
+		"Maintenance is full of equipment that is randomized every round. Look around and see if anything is worth using. There are many places around the ship to hide contraband. A few for starters: linen boxes, toilet cisterns, body bags. Experiment to find more!",
+		"As a Ghost, you can double click on people, bots, or the singularity to follow them.",
+		"Sometimes you won't be able to avoid dying no matter how good you are at the game. it's part of the game, so try not to stress too much about it.",
+		"When a round ends nearly everything about it is lost forever, leave your salt behind with it.",
+		"Sometimes a round will just be a bust. C'est la vie.",
+		"This is a game that is constantly being developed for. Expect things to be added, removed, fixed, and broken on a daily basis.",
+		"Sometimes admins will just do stuff. Roll with it.",
+		"Plenty of things that aren't traditionally considered weapons can still be used to slowly brutalize someone to death, get creative!",
+		"Toolboxes do more damage if they are full of things, but their contents will spill when you use them as a weapon.",
```

Comment:
Does stuff spill out? Never seen that happen.

Reply:
It does.